q2_moshpit/assets/busco/detailed_view.html

```diff
@@ -0,0 +1,258 @@
+{% extends "tabbed.html" %}
+
+{% block head %}
+<title>Embedding Vega-Lite</title>
+<script src="js/bootstrapMagic.js" type="text/javascript"></script>
+<link href="css/styles.css" rel="stylesheet" />
+<script type="text/javascript">
+  // temporary hack to make it look good with Bootstrap 5
+  removeBS3refs();
+</script>
+<script src="https://cdn.jsdelivr.net/npm//vega@5" type="text/javascript"></script>
+<script src="https://cdn.jsdelivr.net/npm//vega-lite@4.17.0" type="text/javascript"></script>
+<script src="https://cdn.jsdelivr.net/npm//vega-embed@6" type="text/javascript"></script>
+<link crossorigin="anonymous" href="https://cdn.jsdelivr.net/npm/bootstrap@5.1.3/dist/css/bootstrap.min.css" integrity="sha256-YvdLHPgkqJ8DVUxjjnGVlMMJtNimJ6dYkowFFvp4kKs=" rel="stylesheet"/>
+{% endblock %}
+
+{% block tabcontent %}
+<script crossorigin="anonymous" integrity="sha256-9SEPo+fwJFpMUet/KACSwO+Z/dKMReF9q4zFhU/fT9M=" src="https://cdn.jsdelivr.net/npm/bootstrap@5.1.3/dist/js/bootstrap.bundle.min.js"></script>
+
+<br>
+<div class="row row-cols-1 row-cols-md-2 g-4">
+  <div class="col-lg-12">
+    <div class="card mt-3 h-10">
+      <h5 class="card-header">Plot description</h5>
+      <div class="card-body">
+        <p>
+          The left plot shows the results generated by BUSCO for <b>all bins</b> and
+          <b> samples</b>. "BUSCO attempts to provide a quantitative assessment
+          of the completeness in terms of the expected gene content of a genome
+          assembly, transcriptome, or annotated gene set. The results are
+          simplified into categories of complete and single-copy, complete and
+          duplicated, fragmented, or missing BUSCOs. BUSCO completeness results
+          make sense only in the context of the biology of your organism". Visit the 
+          <a href="https://busco.ezlab.org/busco_userguide.html#interpreting-the-results">BUSCO User Guide </a>
+          for more information. 
+        </p>
+        <p>
+          The right barplot shows assembly statistics calculated for each bin using BBTools.
+          Specifically, it displays the statistics computed by the <b>stats.sh</b> procedure from BBMap.
+          View the 
+          <a href="https://github.com/BioInfoTools/BBMap/blob/master/sh/stats.sh">
+            source code and documentation 
+          </a>
+          of stats.sh for more information.
+        </p>
+
+        <div style="align-items: center; display: flex">
+          <span class="header-inline">Downloads:</span>
+          <div class="'col-lg-4">
+            <div aria-label="Basic outlined example" class="btn-group" role="group">
```

Comment:
This button is repeated in the table tab. Is that intended?

Reply:
Yes, this is just so that one doesn't need to switch between the tabs to download the TSV.